.eslintrc.js

```diff
@@ -78,6 +78,25 @@ module.exports = {
       rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
         // add your custom rules and overrides for node files here
       })
-    }
+    }, {
+      // mocha tests
+      files: [
+        'node-tests/**/*.js'
+      ],
+      env: {
+        node: true,
+        mocha: true
+      }
+    }, {
+      // ember-cli is guaranteed to be present when running blueprints
+      files: [
+        'blueprints/**/*.js'
+      ],
+      rules: Object.assign({}, require('eslint-plugin-node').configs.recommended.rules, {
+        "node/no-unpublished-require": ["error", {
+          "allowModules": ["ember-cli"]
```

Comment:
Doesn't it seem like this could be / should be part of `'plugin:ember/recommended'`?

Reply:
No, not in `eslint-plugin-ember` -- it's job is just to supply rules for Ember best practices. If anything it would go in `ember-cli`'s addon blueprint (which is what generates this file in addons). But I think this a pretty edgy edge case -- most addon blueprint don't subclass the `Blueprint` class (which is why ours `require` `ember-cli`), but instead do the simpler export of a hash. So I don't think it really makes sense to put extra stuff in every single addon's `eslintrc.js` to address this pretty rare edge case,

Reply:
Gotcha -- that makes me feel better about this `override`.